### ISSUE TITLE
Make DPAD == LeftStick

### ISSUE THREAD

**Issue by Hexaae** [elishacloud/dinputto8#6]
Hope it's not too off-topic...
I've bought a good old arcade joystick 🕹 Hori Fighting Stick Mini recognized by Windows 10 as XInput device with DPAD only for the stick movements.
Since some games don't support DPAD and expect to use only LSTICK, would be possible to create a special version of dinputto8 or a new dinput8.dll wrapper project able to remap DPAD == LSTICK for movements? 😥

**Comment by elishacloud** [comment]
Yes, this is possible.  [Here](https://github.com/ChronoMonochrome/DirectX_Hacks_for_TLoH) is an example of someone who did something like this.  I don't plan to implement this type of feature here, at least not at this time.

But there are already tools that do things like that.  Try using [XInput Plus](https://www.play-old-pc-games.com/compatibility-tools/xinput-plus-tutorial/).

**Comment by ViperAcidZX** [comment]
As much as I love XInput Plus, sometimes its a hit-or-miss for some PC games from my experience. It works great for games like RayStorm and RayCrisis' PC port (though the games themselves have issue with modern Windows support being made around for Windows 95/XP), Touhou 06 ~ The Embodiment of Scarlet Devil (though another game that suffers with modern Windows compatibility issues), and Silent Hill 2 even before the improvements brought by the Enhanced Edition project (albeit without vibration support and limited remapping support at the time), however some games either require a lot of trail-and-error to get them to cooperate with XInput Plus, if at all (e.g. trying to trick OutRun 2006 to think the game is using an XInput pad via a Bluetooth connection), and there are sadly still a good number of games like Mega Man X8 (the original PC version), SUGURI and Acceleration of SUGURI (the non-Steam re-releases), sora and acceleration of SUGURI 2 (ditto), and Streets of Rage Remake (which can't use analog triggers with its limited controller support) to name a few.

**Comment by Hexaae** [comment]
What's the problem with Mega Man X8/X4 PC and XInput Plus? My Hori Fighting Stick Mini 4 joystick wasn't working (the game uses DInput and expected LSTICK for directions) so I remapped LSTICK = DPAD+LSTICK and I solved the problem... XInput worked great!

**Comment by ViperAcidZX** [comment]
Very belated reply but yeah, it works relatively fine with Mega Man X4's PC port, but it doesn't work what so ever with X8 and it's really annoying since I want to not only be able to use the D-pad to move, but also the LT and RT triggers since many old games don't recognize them as buttons on modern XInput controllers.

**Comment by elishacloud** [comment]
Yeah, I have run into a couple games that don't work with dinputto8 yet, I believe they have the same issue, but I have not had time to look into them.  I hope to get to it soon.  I will also check on X8 if I can get a copy of it somewhere.
